classes/controller/AdminController.php

```diff
@@ -1432,8 +1432,8 @@ protected function processUpdateOptions()
                 }
 
                 // Check if field is required
-                if ((!Shop::isFeatureActive() && isset($values['required']) && $values['required'])
-                    || (Shop::isFeatureActive() && isset($_POST['multishopOverrideOption'][$field], $values['required']) && $values['required'])) {
+                if ((!Shop::isFeatureActive() && isset($values['required']) !empty($values['required']))
```

Comment:
Watchout here, you don't remove the `isset($values['required'])`

Reply:
Is it OK now?